src/hooks/useSavingsAccount.js

```diff
@@ -91,19 +91,20 @@ export default function useSavingsAccount(includeDefaultDai) {
     if (!hasAccountAddress) return;
     const fetchBackupSavings = async () => {
       const backup = await getSavings(accountAddress, network);
-      setBackupAccountTokens(backup);
+      if (!isEmpty(backup)) {
+        setBackupSavings(backup);
+      }
     };
     fetchBackupSavings();
   }, [accountAddress, hasAccountAddress, network]);
 
   useEffect(() => {
-    if (!hasAccountAddress || isNil(backupAccountTokens)) return;
+    if (!hasAccountAddress) return;
     const parseSavingsResult = async data => {
       if (error) return;
-      let savingsAccountData = [];
-      const markets = keyBy(data?.markets || [], property('id'));
 
       if (data) {
+        const markets = keyBy(data?.markets || [], property('id'));
```

Comment:
no need to change but i think the keyBy function supports shorthand here like i thiiiiiink it could be keyBy(stuff, ‘id’)

more just sharing weird lodash stuff i’ve recently learned! ❤️

Reply:
good catch! you're right, I updated it